packages/identity/src/odis/identifier-backwards-compatibility.test.ts

```diff
@@ -1,10 +1,10 @@
-import { getPhoneHash } from 'old-celo-base'
 import { soliditySha3 } from '@celo/utils/lib/solidity'
+import { getPhoneHash } from 'old-celo-base'
 import { OdisUtils as OdisUtilsOld } from 'old-identity-sdk'
```

Comment:
When I'm running the test, I'm getting a failing test due to a TS error:

```sh
# building as usual
~/Documents/celo-org/social-connect alecps/fixBackwardsCompTest $ yarn
~/Documents/celo-org/social-connect alecps/fixBackwardsCompTest $ yarn build

# running specific test
~/Documents/celo-org/social-connect/packages/identity alecps/fixBackwardsCompTest $ yarn run test:backwards-compatibility

 FAIL  src/odis/identifier-backwards-compatibility.test.ts
  ● Test suite failed to run

    src/odis/identifier-backwards-compatibility.test.ts:3:43 - error TS2307: Cannot find module 'old-identity-sdk' or its corresponding type declarati
ons.                                 

    3 import { OdisUtils as OdisUtilsOld } from 'old-identity-sdk'
                                                ~~~~~~~~~~~~~~~~~~

Test Suites: 1 failed, 1 total
```

Is this just my setup? Did the test pass on your end @alecps?

Reply:
weird, it is passing for me locally

Reply:
I got the same error after pulling your change that renamed the import to `old-celo-identity`, but it went away when I ran `yarn` again. This makes me think that it might be an issue with the packages on your machine? I'll check with Seb to see if he was able to run it

Reply:
Sounds like Seb was able to run the test, so I'm gonna resolve this